caesar_cypher.cpp

```diff
@@ -1,9 +1,74 @@
 #include <cstring>
+#include <exception>
 #include <iostream>
+#include <regex>
  
-int main()
+const uint8_t CYPHER_SHIFT_KEY_ARGUMENT_INDEX = 1;
+
+uint8_t calculateEncryptedLetterPosition(uint8_t letter, uint32_t shiftKey)
 {
-	std::string inputString;
-	std::getline(std::cin, inputString);
+    if (' ' == letter)
+    {
+        return (uint8_t)letter;
+    }
+
+    if (('A' <= letter && 'Z' >= letter) ||
+        ('a' <= letter && 'z' >= letter))
+    {
+        const uint8_t alphabetASCIIStarterPosition = ('Z' >= letter) ? ('A' - 1) : ('a' - 1);
```

Comment:
`const uint8_t alphabetAsciiStarterPosition = 'Z' >= letter ? 'A' : 'a' - 1;`

Reply:
Well, here I just don't need to type parentheses?